extensions/micrometer/deployment/src/test/java/io/quarkus/micrometer/deployment/binder/NettyMetricsTest.java

```diff
@@ -0,0 +1,45 @@
+package io.quarkus.micrometer.deployment.binder;
+
+import jakarta.enterprise.inject.Any;
+import jakarta.enterprise.inject.Instance;
+import jakarta.inject.Inject;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import io.micrometer.core.instrument.binder.MeterBinder;
+import io.micrometer.core.instrument.binder.netty4.NettyAllocatorMetrics;
+import io.micrometer.core.instrument.binder.netty4.NettyEventExecutorMetrics;
+import io.quarkus.test.QuarkusUnitTest;
+
+public class NettyMetricsTest {
```

Comment:
We should have tests that cover each one of the classes being instrumented...

Reply:
Which classes and instrumented how?

Reply:
How do you know you are getting metrics from the pools and they reflect correct values?
By creating requests on the right places, it should be possible to assert the executor metrics, no?

Reply:
> How do you know you are getting metrics from the pools and they reflect correct values?

Atm I trust the Micormeter code do do that.

Let me try and add something ...

Reply:
Thanks @alesj, @franz1981